torch/onnx/errors.py

```diff
@@ -46,6 +45,9 @@ class UnsupportedOperatorError(OnnxExporterError):
     """Raised when an operator is unsupported by the exporter."""
 
     def __init__(self, name: str, version: int, supported_version: Optional[int]):
+        # Delayed import to avoid circular dependency.
+        from torch.onnx._internal import diagnostics
```

Comment:
maybe needed because mypy?
```suggestion
        from torch.onnx._internal import diagnostics  # type: ignore[import]
```

Reply:
mypy has no complaints right now?

Reply:
Didn't see complaints from lintrunner locally. Should be fine unless later torch CI says no.

Reply:
no, and that was funny. It usually complains about `import` not on the beginning of the file